### DISCUSSION TITLE
archives/Hexo/7648

### DISCUSSION THREAD

**Discussion by giscus** [4rozeN/4rozeN.github.io#7] [General]
# archives/Hexo/7648

Hexo 博客配置 GitHub Actions 推送 indexnow    什么是 indexnow？   API Key 找到 Indexnow 官网生成 API Key（每点击一次 Generate 都会生成一个新的 Key）：  你需要点击下载按钮将该 5617c6a058f84405b90e2a0c6e99519a.txt 文件保存好。 然后把它放到你的网站根目录下，让它可以通过

https://4rozen.github.io/archives/Hexo/7648.html

**Comment by MasoFod** [comment]
非常有用，感谢！

**Comment by Jingqing3948** [comment]
感谢。
